src/api/Auth.ts

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import axios from "axios";
+import { BASE_URL } from "../constant";
```

Comment:
`constant` is a module registered both with `tsconfig` and `babel.config`. You can directly use 
```ts
import { BASE_URL } from "constant";
```